anyway/models.py

```diff
@@ -3081,3 +3081,44 @@ class TelegramForwardedMessages(Base):
     message_id = Column(String(), primary_key=True)
     newsflash_id = Column(BigInteger(), nullable=False)
     group_sent = Column(String(), nullable=False)
+
+
+class SDInvolved(Base):
```

Comment:
`ForeignKeyConstraint` should be added to table model class, [just like in Involved table](https://github.com/data-for-change/anyway/blob/93c1d1d59dc43bf03cf4df171513ebb4c4a61ea8/anyway/models.py#L800)

Reply:
And perhaps also `ForeignKey` [like in `LocationVefiricationHistory`](https://github.com/data-for-change/anyway/blob/93c1d1d59dc43bf03cf4df171513ebb4c4a61ea8/anyway/models.py#L1020)